Automattic/jetpack
# VideoPress: Allow keyboard navigation on video quick actions

Fixes #27331

#### Changes proposed in this Pull Request:

* On VideoCard component, show the card if the focus is within the card
* On VideoRow component, change the quick actions rendering mechanism from state-based to CSS-based to leverage both `:hover` and `focus-within` capabilities in order to allow keyboard navigation

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard with at least one video
* Use the `tab` key to navigate the video actions without using the cursor, both on Grid and List layouts
* Check that there is no other side-effect in the page layout

https://user-images.githubusercontent.com/8486249/201402513-43e0a479-717a-48b3-8383-d70fb7936b91.mp4
